### ISSUE TITLE
Brave search to default browsers

### ISSUE THREAD

**Issue by CyberaxAnnon** [bromite/bromite#2080]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md)
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have searched [existing issues](https://github.com/bromite/bromite/issues) for my feature request. This is a new issue (NOT a duplicate) and is not related to another issue.

### Is your feature request related to privacy?

Yes

### Is there a patch available for this feature somewhere?

No

### Describe the solution you would like

As described in the title

### Describe alternatives you have considered

Other browsers

**Comment by n-buna404** [comment]
> Can you add this search engine as default?

No. Bromite does not make any choice related to default search engines, the Chromium default is used. Various Android browsers get some fee to ship their apps with a specific default search engine, Bromite does not get any fee from anyone. Changing the default search engine would lead to an endless series of requests to change it based on personal preferences, thus no change is made to the default. See also: https://github.com/bromite/bromite/wiki/SearchEngines

Please really read FAQs before opening issue.
